overblog/graphiql-bundle/0.1/post-install.txt

```diff
@@ -0,0 +1,3 @@
+<bg=blue;fg=white> Next for OverblogGraphiQLBundle </>
+    1. The bundle is auto configured now for <comment>dev</comment> environment only
```

Comment:
I would still remove this line which does not say anything useful.

Reply:
Sure!

Reply:
@fabpot done